scripts/aws_replicate.py

```diff
@@ -18,14 +18,19 @@
 import boto3
 import botocore
 
+try:
+    from urllib.parse import urlparse
+except ImportError:# Python 2.x
+    from urlparse import urlparse
```

Comment:
you are not trying to be py2 compatible (since you removed python 2 testing from travis) so you can just use `from urllib.parse import urlparse`